CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 Changelog
 =========
 
+## [1.0.0](https://github.com/ckeditor/ckeditor5-integrations-common/compare/v0.2.1...v1.0.0) (2024-09-03)
+
+### Features
```

Comment:
```suggestion

We are happy to announce the first stable release of `@ckeditor/ckeditor5-integrations-common`, a package containing utility modules for integrating with CKEditor 5.

### Features
```

I think we miss in README the general purposes of the package and explaining what it offers and why developers need to use it. I do not want to block the release, but consider extending README.